src/components/PopOver/index.js

```diff
@@ -0,0 +1,240 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { themes, constants, spacing } from "../../theme";
+
+const StyledPopOver = styled.div`
+  background-color: ${themes.global.white.base};
+  border: 1px solid ${themes.global.gray02};
+  border-radius: ${constants.borderRadius.large};
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.12);
+  position: absolute;
+  max-width: 260px;
+  padding: ${spacing.moderate};
+  display: ${({ isVisible }) => (isVisible ? "block" : "none")};
+  top: ${({ top }) => `${top}px`};
+  left: ${({ left }) => `${left}px`};
+`;
+
+const MIN_SPACE_FROM_EDGE = 24;
+const MOBILE_MIN_SPACE_FROM_EDGE = 16;
+const MOBILE_MAX_WIDTH = 767;
+const SPACE_FROM_MOUSE = 10;
+
+class PopOver extends Component {
```

Comment:
I see that there is a mock which uses the `PopOver` as a tooltip or an overflow block. Are there any other use cases that you had in mind when building this component?

Reply:
PopOver allows the developer to decide if it should render on the view port or it should be inside a container and visible on the view port. It should appear on bottom if possible if not it should appear on top. It also allows the developer to configure sticky header/footer which shrinks view port.

The difference between PopOver and tooltip is that the tooltip can appear on the left and right side as well. Tooltip also has an arrow pointing to the element that triggered the tooltip.

Reply:
Thank you for the explanation. Sorry I didn't ready your PR message in great enough detail.